datasets/sentinel-1-rtc/s1rtc.py

```diff
@@ -57,11 +55,12 @@
 }
 
 
-class S1RTCCollection(Collection):
+class S1RTCCollection(Collection):  # type: ignore
```

Comment:
Out of curiosity, why the need for this `type: ignore`?

Reply:
I'm not sure. I was getting a typing error, but no longer see it. I'll clean those up.